src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -48,7 +48,7 @@ export const RadioGroup = <T, >({
     return options.map((option, index) => (
       <Radio
         description={option.description}
-        idDisabled={option.disabled}
+        isDisabled={option.disabled}
```

Comment:
nice catch! please highlight this in the PR title so that it gets proper notice in the generated github release

Also I'm adding the bug label for this

Reply:
I think the typo is just on the radio button that is not exported by the library or meant to be used alone, so probably it's not required (it was correctly valued from the key "disabled" of the RadioGroupOptions)